develop/buildkite/pipeline.yml

```diff
@@ -306,21 +325,37 @@ steps:
           run: integration-test-sqlite
           config: ./develop/buildkite/docker-compose.yml
 
-  - wait
+  - wait: ~
+    continue_on_failure: true  # required for test reports of failed build to be picked up
+  
+  - label: ":mag: test-report"
+    agents:
+      queue: "default"
+      docker: "*"
+    command: buildkite-agent artifact download ".testreport/*.xml" .
+    retry:
+      automatic:
+        limit: 2
+    plugins:
+      - test-collector#v1.10.0:
+          files: ".testreport/*.xml"
+          format: junit
 
   - label: ":coverage: coverage-report"
     agents:
       queue: "default"
       docker: "*"
-    command: "./develop/buildkite/scripts/coverage-report.sh"
+    command: "make ci-coverage-report"
```

Comment:
one small thought: this could check if there are zero files in `.coverage` and abort if so. we don't know how many there should be but zero is definitely wrong. that would skip this on totally broken builds.

no need to do it in this pr though (or at all really)

Reply:
Done!